README.md

```diff
@@ -12,62 +12,361 @@ Install from NPM:
+  const elem = await fixture(html`<my-elem></my-elem>`, {
+    viewport: { height: 300, width: 200 }
+  });
+  // do assertions
+});
+```
+
+#### Configuring the Language or Text Direction
+
+If the component under test has special multi-lingual or bidirectional text behavior, both `language` and `rtl` (right-to-left) options are available.
+
+```javascript
+it('should work in French', async() => {
+  const elem = await fixture(html`<my-elem></my-elem>`, {
+    lang: 'fr'
+  });
+  // do assertions
+});
+
+it('should work in RTL', async() => {
+  const elem = await fixture(html`<my-elem></my-elem>`, {
+    rtl: true
+  });
+  // do assertions
+});
+```
+
+If `lang` is set to Arabic (`ar`), the right-to-left option will automatically be enabled.
+
+> **Note:** it's not recommended to use `language` configuration with visual-diff to solely test the correctness of translations. The [messageformat-validator](https://github.com/bearfriend/messageformat-validator) is a more efficient way to test translations.
+
+### Accessibility Testing with aXe
+
+Elements can be processed by the [aXe accessibility validator](https://github.com/dequelabs/axe-core), which will automatically fail the test if any violations are detected.
+
+```javascript
+it('should be accessible', async() => {
+  const elem = await fixture(html`<button></button>`);
+  // will fail since the button is not labelled
+  await expect(elem).to.be.accessible();
+})
+```
+
+> **Important:** the call to `to.be.accessible()` is asynchronous -- don't forget to `await` it!
+
+### Using the Mouse
+
+To hover over or click on a specific element, use `hoverElem(elem)` and `clickElem(elem)`.
+
+```javascript
+import { clickElem, fixture, hoverElem } from '@brightspace-ui/testing';
+
+it('should hover over element', async() => {
+  const elem = await fixture(...);
+  await hoverElem(elem);
+  // do assertions
+});
+
+it('should click on element', async() => {
+  const elem = await fixture(...);
+  await clickElem(elem);
+  // do assertions
+});
+```
+
+Alternatively, to hover over or click at viewport coordinates with the mouse, use `hoverAt(x, y)` or `clickAt(x, y)`:
+
+```javascript
+import { clickAt, hoverAt } from '@brightspace-ui/testing';
+
+it('should hover at coordinate', async() => {
+  await hoverAt(100, 200);
+  // do assertions
+});
+
+it('should click at coordinate', async() => {
+  await clickAt(100, 200);
+  // do assertions
+});
+```
+
+### Using the Keyboard
+
+To place focus on an element using the keyboard, use `focusElem(elem)`. Doing so will trigger its `:focus-visible` CSS pseudo-class.
+
+```javascript
+import { fixture, focusElem } from '@brightspace-ui/testing';
+
+it('should focus on element', async() => {
+  const elem = await fixture(html`<button>elem</button>`);
+  await focusElem(elem);
+  // do assertions
+});
+```
+
+To send particular keystrokes to the browser window or a specific element, use `sendKeys(action, keys)` or `sendKeysElem(elem, action, keys)`.
```

Comment:
I think it'd be worth calling out somewhere in here that `sendKeysElem` will focus the element first, not just fire a keyboard event from it